bench-exchange/src/main.rs

```diff
@@ -66,7 +46,40 @@ fn main() {
         batch_size,
         chunk_size,
         account_groups,
+        client_ids_and_stake_file,
+        write_to_client_file,
+        read_from_client_file,
     };
 
-    do_bench_exchange(vec![client], config);
+    if !write_to_client_file {
```

Comment:
This if/else could return the client vector and then unify the call to `do_bench_exchange`?

Reply:
👍